2020/_data/speakers.json

```diff
@@ -1,61 +1,131 @@
 {
   "items": [
     {
-      "name": "Venkat Subramaniam",
-      "description": "Java Champion, Founder of Agile Developer, Inc.",
-      "biography": "Dr. Venkat Subramaniam is an award-winning author, founder of Agile Developer, Inc., creator of agilelearner.com, and an instructional professor at the University of Houston. He has trained and mentored thousands of software developers in the US, Canada, Europe, and Asia, and is a regularly-invited speaker at several international conferences. Venkat helps his clients effectively apply and succeed with sustainable agile practices on their software projects. Venkat is a (co)author of multiple technical books, including the 2007 Jolt Productivity award winning book Practices of an Agile Developer. You can find a list of his books at agiledeveloper.com. You can reach him by email at venkats@agiledeveloper.com or on twitter at @venkat_s.",
       "enabled": "1",
-      "homepage": "http://agiledeveloper.com",
-      "image": "assets/img/speakers/venkat-subramaniam.jpg",
-      "ref": "VmVua2F0U3VicmFtYW5pYW12ZW5rYXRzQGFnaWxlZGV2ZWxvcGVyLmNvbQ",
-      "url": "infoSpeaker.html?ref=VmVua2F0U3VicmFtYW5pYW12ZW5rYXRzQGFnaWxlZGV2ZWxvcGVyLmNvbQ",
-      "twitter": "@venkat_s"
+      "name": "Ixchel Ruiz",
+      "company": "Karakun AG",
+      "description": "",
```

Comment:
We don't have descriptions anymore?

Reply:
![image](https://user-images.githubusercontent.com/100741/73496466-f1eb6200-43b8-11ea-8df8-a39766d3bc7b.png)
added them now

Reply:
la photo de Vaughn es la de este año?

Reply:
ahora si
![image](https://user-images.githubusercontent.com/100741/73610983-774f5c00-45dd-11ea-914d-f433457465ea.png)
estos tres también los actualicé
![image](https://user-images.githubusercontent.com/100741/73610990-90580d00-45dd-11ea-8590-bdf5787d0d42.png)